Client/src/App.tsx

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
+import { Route,Routes } from 'react-router-dom';
 
 
 function App() {
   return (
     <div>
-      <h1>Hello</h1>
+      <Routes>
```

Comment:
you need to put path in routes and create pages and import it into app.tsx and put those components as elements